crd/app-crd.yaml

```diff
@@ -19,12 +19,13 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: applications.app.k8s.io
   annotations:
-    api-approved.kubernetes.io: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/2
+    api-approved.kubernetes.io: unapproved, for Google Cloud Marketplace use only
```

Comment:
Following pattern in https://github.com/kubernetes/kubernetes/pull/101871#discussion_r631184970